tsconfig.lint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+  },
+  // NOTE (from https://www.typescriptlang.org/docs/handbook/tsconfig-json.html):
```

Comment:
I don't think JSON supports comments.

Reply:
Strict JSON doesn't, but tsc explicitly does. In fact the `tsconfig.json` file it creates when running `tsc --init` contains most of the options it supports commented out.
I could remove the comment, but I think the behavior obtained through the absence of some fields is worth a note.